Marlin/pins_BQ_ZUM_MEGA_3D.h

```diff
@@ -60,19 +60,34 @@
 #define PS_ON_PIN         81 // External Power Supply
 
 #if ENABLED(BQ_LCD_SMART_CONTROLLER) // Most similar to REPRAP_DISCOUNT_SMART_CONTROLLER
```

Comment:
I'm thinking this block should just move to `pins_RAMPS_14.h` with the rest of the LCD controller pins.